### DISCUSSION TITLE
Select with option groups, filtering and keyboard navigation?

### DISCUSSION THREAD

**Discussion by jammy-git** [quasarframework/quasar#10278] [General - Components / Directives / etc]
I have a select set up with both option groups (utilising the option slot) and filtering. However this does not work with keyboard navigation and feels a little "hacky". Is there a more robust solution to building up a select dropdown with option groups that still maintains filtering and keyboard navigation?

**Comment by pdanpdan** [comment]
No, sorry, there isn't for the moment.

Just linking to the feature request: https://github.com/quasarframework/quasar/issues/5724

Can you make a codepen with your implementation of groups, maybe we can play with it a little.
